### DISCUSSION TITLE
Submit flatpak to Flathub?

### DISCUSSION THREAD

**Discussion by tulilirockz** [input-leap/input-leap#2139] [Feature Requests]
### What happened?

Would there be any chance we could get this flatpak on flathub? The builds seem to work so I dont really see why we wouldnt be able to add it to flathub too

### Version

v3.0.2

### If applicable, where did you install Input Leap from?

-

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### If applicable, are you using Wayland or X11?

Wayland

### What OS versions are you using?

Flatpak Freedesktop Runtime

**Comment by tulilirockz** [comment]
Would there be any interest in that? I would love to try to submit it

**Comment by Justinzobel** [comment]
Also willing to help with getting Input Leap on Flathub.

**Reply by tulilirockz** [reply]
Everything on this repo seems to be up to flathub standards, I believe the only thing missing would be to just submit it! - Maybe a few changes to the manifest, but thats it!

**Reply by tulilirockz** [reply]
Nope, the manifest needs a lot of work still
